tracer-core/src/main/java/org/zalando/tracer/FlowIDGenerator.java

```diff
@@ -15,15 +16,15 @@
 
     @Override
     public String generate() {
-        final UUID uuid = UUID.randomUUID();
+        final Random random = ThreadLocalRandom.current();
```

Comment:
What about `return new UUID(random.nextLong(), random.nextLong()`?

Reply:
Not all bits of a UUID are random and two random longs are not guaranteed to be a valid UUID.

Reply:
Also, why constructing a UUID from two longs if I'm extracting those longs 2 lines down?